src/main/java/marquez/MarquezContext.java

```diff
@@ -134,49 +153,17 @@ private MarquezContext(
             jobResource,
             tagResource,
             serviceExceptionMapper,
-            jdbiException);
+            jdbiException,
+            openLineageResource);
   }
 
-  public void registerListener(@NonNull RunTransitionListener listener) {
-    runTransitionListeners.add(listener);
-  }
-
-  public static Builder builder() {
-    return new Builder();
-  }
+  public ObjectMapper getLineageObjectMapper() {
```

Comment:
I would move creating a new instance of `ObjectMapper` for _OpenLineage_ to [`Utils`](https://github.com/MarquezProject/marquez/blob/main/src/main/java/marquez/common/Utils.java) as we keep things shared across boundaries (_resource_, _service_, etc) related to (de)serialization, etc under the `common` pkg.

We can also be explicit here and use the full name for _OpenLineage_ :)

```
Utils.newOpenLineageObjectMapper()
```